apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLoggerExtension.cs

```diff
@@ -20,7 +20,13 @@
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using System;
 
+#if NETCOREAPP3_1
```

Comment:
Do we have to target netcoreapp3.1 instead of netstandard2.1? That wouldn't entirely surprise me, but I thought I'd check. (It looks a little odd as a condition here.)

Reply:
Yep, I started with netstandard2.1, but we really need to target netcoreapp3*: https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#migrate-libraries-via-multi-targeting